genesyscloud/resource_genesyscloud_architect_ivr.go

```diff
@@ -327,3 +370,27 @@ func flattenIvrDnis(dnisArr *[]string) *schema.Set {
 	}
 	return nil
 }
+
+func uploadChunk(d *schema.ResourceData, chunk []string, architectApi *platformclientv2.ArchitectApi) diag.Diagnostics {
+	var dnis []string
+
+	ivr, _, getErr := architectApi.GetArchitectIvr(d.Id())
```

Comment:
It would be cool if we could proxy out the GetArchitectIvr so that we could test this one function without any other dependencies.